components/aws/sagemaker/deploy/component.yaml

```diff
@@ -1,147 +1,126 @@
+    (EI) instance to use for the production variant., default: ''}
+- {name: variant_name_3, type: String, description: The name of the production variant.,
+  default: variant-name-3}
+- {name: model_name_3, type: String, description: The model name used for endpoint
+    deployment., default: ''}
+- {name: initial_instance_count_3, type: Integer, description: Number of instances
+    to launch initially., default: '1'}
+- {name: instance_type_3, type: String, description: The ML compute instance type.,
+  default: ml.m4.xlarge}
+- {name: initial_variant_weight_3, type: String, description: Determines initial traffic
+    distribution among all of the models that you specify in the endpoint configuration.,
+  default: '1.0'}
+- {name: accelerator_type_3, type: String, description: The size of the Elastic Inference
+    (EI) instance to use for the production variant., default: ''}
 outputs:
-  - {name: endpoint_name,          description: 'Endpoint name'}
+- {name: endpoint_name, description: The created endpoint name.}
 implementation:
   container:
-    image: amazon/aws-sagemaker-kfp-components:0.9.0
-    command: ['python3']
-    args: [
-      deploy.py,
-      --region, {inputValue: region},
-      --endpoint_url, {inputValue: endpoint_url},
-      --assume_role, {inputValue: assume_role},
-      --endpoint_config_name, {inputValue: endpoint_config_name},
-      --variant_name_1,{inputValue: variant_name_1},
-      --model_name_1, {inputValue: model_name_1},
-      --initial_instance_count_1, {inputValue: initial_instance_count_1},
-      --instance_type_1, {inputValue: instance_type_1},
-      --initial_variant_weight_1, {inputValue: initial_variant_weight_1},
-      --accelerator_type_1, {inputValue: accelerator_type_1},
-      --variant_name_2,{inputValue: variant_name_2},
-      --model_name_2, {inputValue: model_name_2},
-      --initial_instance_count_2, {inputValue: initial_instance_count_2},
-      --instance_type_2, {inputValue: instance_type_2},
-      --initial_variant_weight_2, {inputValue: initial_variant_weight_2},
-      --accelerator_type_2, {inputValue: accelerator_type_2},
-      --variant_name_3,{inputValue: variant_name_3},
-      --model_name_3, {inputValue: model_name_3},
-      --initial_instance_count_3, {inputValue: initial_instance_count_3},
-      --instance_type_3, {inputValue: instance_type_3},
-      --initial_variant_weight_3, {inputValue: initial_variant_weight_3},
-      --accelerator_type_3, {inputValue: accelerator_type_3},
-      --resource_encryption_key, {inputValue: resource_encryption_key},
-      --endpoint_config_tags, {inputValue: endpoint_config_tags},
-      --endpoint_name, {inputValue: endpoint_name},
-      --endpoint_tags, {inputValue: endpoint_tags},
-      --update_endpoint, {inputValue: update_endpoint},
-      --endpoint_name_output_path, {outputPath: endpoint_name}
-    ]
+    image: amazon/aws-sagemaker-kfp-components:0.8.0
```

Comment:
update to latest "0.9.0"

Reply:
Will do a release process before I merge, so this will be updated soon.